benchmarks/asv_numpy.conf.json

```diff
@@ -16,18 +16,22 @@
     // Customizable commands for building, installing, and
     // uninstalling the project. See asv.conf.json documentation.
     //
-    "install_command": [
-        "pip install torch torchvision",
-        "in-dir={env_dir} python -mpip install {wheel_file}"],
-    // "uninstall_command": ["return-code=any python -mpip uninstall -y {project}"],
+    //"install_command": [
```

Comment:
Can we remove all these commented-out lines? I don't see why we need them.

Reply:
done (normally, this config file is automatically created and those comments are still inserted)